src/styles/components/_Button.scss

```diff
@@ -27,25 +27,25 @@
     border-color: transparent;
   }
 
-  @if $ds-include-focus-styles {
-    &:focus,
-    &.ds-c-button--focus {
-      background-color: $focus-color;
-      border-color: transparent;
-      color: $color-base;
-    }
+  // @if $ds-include-focus-styles {
```

Comment:
[question/suggestion] Are these styles still necessary for the future? If not, consider removing.